src/utils/scroll.ts

```diff
@@ -30,6 +30,17 @@ export const isVisibleInContainer = (element: HTMLElement, container: HTMLElemen
 	return elementRect.bottom <= containerRect.bottom && elementRect.top >= containerRect.top;
 };
 
+export const getScrollableParents = (node: HTMLElement) => {
+	const elements = [document.body];
```

Comment:
Check if we're in the browser to use `document`?

Reply:
Shouldn't be necessary given that we only query when the components have live DOM ... Really we need a proper strategy to make the components SSR safe, as-is there's a few that won't even initialize